yt/data_objects/grid_patch.py

```diff
@@ -214,7 +214,7 @@ def child_mask(self):
         for child in self.Children:
             self._fill_child_mask(child, child_mask, 0)
         for sibling in self.OverlappingSiblings or []:
-            self._fill_child_mask(sibling, child_mask, 0)
+            self._fill_child_mask(sibling, child_mask, 0, dlevel=0)
```

Comment:
while we are at it, it would be clearer to use full explicit keywords here, though it's not required for merging this.
```suggestion
            self._fill_child_mask(child=sibling, mask=child_mask, tofill=0, dlevel=0)
```